src/Text/Pandoc/Readers/Markdown.hs

```diff
@@ -1822,13 +1822,35 @@ parenthesizedChars = do
   result <- charsInBalanced '(' ')' litChar
   return $ "(" <> result <> ")"
 
+pBase64DataURI :: PandocMonad m => MarkdownParser m Text
+pBase64DataURI = mconcat <$> sequence
+  [ (textStr "data:")
+  , (T.singleton <$> (letter <|> digit))
+  , restrictedName
+  , (T.singleton <$> char '/')
+  , restrictedName
+  , textStr ";" <* trace "cool"
+  , (mconcat <$> many mediaParam)
+  , textStr "base64," <* trace "fine"
+  , parseBase64String
```

Comment:
What if we replace l.1835 with
```
   many1Char (satisfy (A.inClass "A-Za-z0-9+/"))
,  manyChar (char '=')
```
and get rid of the special T.P.Parsing.Base64 module.  What is the impact on performance on the sorts of files that were problematic before?  I'd like to try the simplest possible thing before worrying about possible optimizations.